src/libs/scene_utils/README.md

```diff
@@ -0,0 +1,88 @@
+# IoT TwinMaker Scene SDK
+
+## Use
+
+This package provides sample command line scripts that automate the creation of IoT TwinMaker scenes. Upload 3D assets and place them in a 3D scene connected to IoT data via augmented widgets (tag, motion indicator, etc.).
+
+## Install
```

Comment:
from the README it might not be obvious to customers they need to set the environment variables

so may be good to have a step here to run `npx ts-node samples/cookie_factory_sample/sample.ts -h` to guide them on how to setup their environment before running the example command (I just saw `(node:96405) UnhandledPromiseRejectionWarning: Error: ConfigError: Missing region in config` which doesn't mention environment variables for example)

Reply:
Will do!